### PR TITLE
Remove apostrophe

### PR DESCRIPTION
**Pull request by Skovy**
Remove an incorrect apostrophe from the `printInclusive` documentation and update the sentence.

### CODE REVIEW

File: docs/docs/addons-perf.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ Get the opaque data structure describing measurements from the last start-stop s
 Perf.printInclusive(measurements)
 ```
 
-Prints the overall time taken. If no argument's passed, defaults to all the measurements from the last recording. This prints a nicely formatted table in the console, like so:
+Prints the overall time taken. If no arguments passed, defaults to all the measurements from the last recording. This prints a nicely formatted table in the console, like so:
```
Comment (jquense) [inline docs/docs/addons-perf.md]: this is an awkward sentance...maybe "When no arguments are passed, `printInclusive` defaults ..."?

**Comment by Skovy** [comment]
Thanks @montogeek and @jquense! I updated based off your suggestion 👍

**Comment by gaearon** [comment]
This is fine, thanks! The addon itself is not going to work in React 16 though 😞

**Comment by Skovy** [comment]
@gaearon really?! 😞  why won't the addon work in React 16? Is it something that would be fixable?

**Comment by gaearon** [comment]
Please check the release notes. The blog post mentions this.

https://facebook.github.io/react/blog/2017/09/26/react-v16.0.html

**Pull request merged**